Source/WebCore/css/CSSSelector.h

```diff
@@ -23,46 +23,53 @@
 
 #include "QualifiedName.h"
 #include "RenderStyleConstants.h"
+#include <wtf/FixedVector.h>
 
 namespace WebCore {
-    class CSSSelectorList;
+
+class CSSSelectorList;
+
+struct PossiblyQuotedIdentifier {
+    AtomString identifier;
+    bool wasQuoted { false };
```

Comment:
Maybe we can use an enum class? e.g. `enum class QuoteState { NotQuoted, Quoted }`.

Reply:
I’m not sure that would improve things. We normally do this to avoid passing a constant true or false in a context where it’s not clear what it means. But in this case there is nowhere doing that. There is only one call site passing this value and it passes the result of a boolean expression.

However if you are convinced it would make the code better I would do it.

Reply:
Okay. I don't feel strongly about it either way.